run_backend.sh

```diff
@@ -7,6 +7,7 @@ Usage: run_server [-c <file>] [-t <file>]
 
 -c | -config <file>: SPIRE Config File
 -t | -tornjak-config <file>: Tornjak Config File
+-expandEnv: expand environment variables expressed in config files
```

Comment:
Is this a key=value pair? Can you provide a sample example here? Does it take a single or multiple env. variable. If multiple, what is the format? Thank you!

Reply:
i have edited the usage documenation in the `docs/config-tornjak-agent.md` file. Maybe this should be a different file name?